astronomer/airflow/version_check/models.py

```diff
@@ -41,7 +41,9 @@ def ensure_singleton(cls):
                 session.rollback()
 
     @classmethod
-    def acquire_lock(cls, check_interval, session):  # type: (datetime.timedelta, sqlalchemy.Session) -> Optional[AstronomerVersionCheck]
+    def acquire_lock(
+        cls, check_interval, session
+    ):  # type: (datetime.timedelta, sqlalchemy.Session) -> Optional[AstronomerVersionCheck]
```

Comment:
You can put the type comment on the next line:

```python
def acquire_lock(cls, check_interval, session):
    # type: (datetime.timedelta, sqlalchemy.Session) -> Optional[AstronomerVersionCheck]
```

Probably looks better? It’s subjective.